src/pages/Error.tsx

```diff
@@ -1,12 +1,14 @@
-import { Alert, AlertTitle } from '@mui/material';
+import { Alert, AlertTitle, Box } from '@mui/material';
 import PageContainer from 'components/shared/PageContainer';
+import { NavLink } from 'react-router-dom';
 
 const Error = () => {
     return (
         <PageContainer>
             <Alert severity="error">
                 <AlertTitle>Error</AlertTitle>
-                This path does not exist.
+                <Box>This path does not exist.</Box>
+                <NavLink to="/">Return to homepage</NavLink>
```

Comment:
Added a link on this page so folks don't get stuck while we dev on a Staging environment.